src/migrations/annotation.test.ts

```diff
@@ -0,0 +1,47 @@
+import { defaultQuery, EditorMode } from 'types';
+import { migrateAnnotation } from './annotation';
+
+const OLD_ANNOTATION = {
+  database: 'PerfTest',
+  datasource: 'Azure Data Explorer Datasource',
+  enable: true,
+  iconColor: 'red',
+  name: 'New annotation',
+  query: 'PerfTest\n| where $__timeFilter(_Timestamp_) and _val1_ > 15\n| project _Timestamp_, Text=_flag1_',
+  resultFormat: 'table',
+};
+
+const NEW_ANNOTATION = {
+  database: 'PerfTest',
+  datasource: 'Azure Data Explorer Datasource',
+  enable: true,
+  iconColor: 'red',
+  name: 'New annotation',
+  query: 'PerfTest\n| where $__timeFilter(_Timestamp_) and _val1_ > 15\n| project _Timestamp_, Text=_flag1_',
+  resultFormat: 'table',
+  target: {
+    database: 'PerfTest',
+    expression: defaultQuery.expression,
+    pluginVersion: defaultQuery.pluginVersion,
+    query: 'PerfTest\n| where $__timeFilter(_Timestamp_) and _val1_ > 15\n| project _Timestamp_, Text=_flag1_',
+    querySource: EditorMode.Raw,
+    rawMode: true,
+    refId: 'Anno',
+    resultFormat: 'table',
+  },
+};
+
+describe('ADX: migrateAnnotation', () => {
```

Comment:
minor nit: everything in these tests are "ADX", so it probably doesnt need the prefix 😅

Reply:
done